data_checker/main.resource_policy_check.py

```diff
@@ -0,0 +1,134 @@
+    result_t = cursor.fetchall()
+    cursor.close()
+    conn.close()
+    # create list from select result
+    for resource_id_t in result_t:
+        resource_id = resource_id_t[0]
+        resource_ids_list.append(resource_id)
+    return resource_ids_list
+
+if __name__ == "__main__":
+    _logger.info('Resource policies checker for anonymous view of items')
+    item_dict_json = "item_dict.json"
+    handle_json = "handle.json"
+    # get a dictionary mapping dspace5 IDs to dspace7 IDs for items
+    item_dict = create_dict_from_json(item_dict_json)
+    # get IDs of item from dspace5 base od select
+    old_item_list = get_data_from_database()
+    # get IDs for dspace7 from IDs from dspace5 based on map
+    new_item_list = convert_old_ids_to_new(old_item_list, item_dict)
+    # list od item IDs from dspace7 which can READ Anonymous
+    item_ids_list=[]
+    # get total pages for search
+    # max page size for this request is 100
+    response = rest_proxy.get('discover/search/objects?sort=score,'
+                              'DESC&size=100&page=0&configuration=default'
+                              '&dsoType=ITEM&embed=thumbnail&embed=item%2Fthumbnail')
+    response_json = convert_response_to_json(response)
+    totalPages = objects = response_json['_embedded']['searchResult']['page']['totalPages']
+    # get result from each page
+    # we don't get items which are withdrawn or discoverable
+    for page in range(totalPages):
+        response = rest_proxy.get('discover/search/objects?sort=score,DESC&size=100&page=' +
+                                  str(page) +
+                                  '&configuration=default&'
+                                  'dsoType=ITEM&embed=thumbnail&embed=item%2Fthumbnail')
+        response_json = convert_response_to_json(response)
+        objects = response_json['_embedded']['searchResult']['_embedded']['objects']
+        # add each object to result list
+        for item in objects:
+            item_ids_list.append(item['_embedded']['indexableObject']['id'])
+
+    # compare expected items in dspace5 and got items from dspace7
+    # log items, which we cannot find
+    item_url = 'core/items'
+    for id_ in new_item_list:
+        if id_ in item_ids_list:
+            item_ids_list.remove(id_)
+        else:
+            # check if we really don't have access to item in Dspace7
+            response = do_api_get_one(item_url, id_)
+            if not response.ok:
+                _logger.error('Do not exist resource policy of anonymous READ for item: ' +
+                              id_ + ' in dspace7!')
+    #now in new_item_list are items whose resource_policy
+    # was not found in dspace5
+    # it could be because in dspace7 is using inheritance for resource policies
+    # check if you have access for these items in dspace5
+    # based on their handles or there was import error
+    item_lindat_url = 'https://lindat.mff.cuni.cz/repository/xmlui/handle/'

```

Comment:
this should be in `const.py`, no?

Reply:
no, I do it like this in other files as well